aict_tools/scripts/fact_to_dl3.py

```diff
@@ -381,7 +381,7 @@ def main(
         sample_fraction = f.attrs.get('sample_fraction', 1.0)
 
     set_sample_fraction(output, sample_fraction)
-    copy_runs_group(data_path, output)
+    copy_group(data_path, output, 'runs')
```

Comment:
the function you changed before also copied the `corsika_runs` group.

Reply:
Whoops.
This can probably be reversed any way if I remember correctly

Reply:
Or just call it twice

Reply:
If we like this version more, sure.